lib/engine/g_1856/corporation.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require_relative '../corporation'
```

Comment:
is it really necessary to have a new corp type? can these methods just be on game?

Reply:
I'm not super familiar with ruby so it's completely possible there are things that Ruby supports that I don't realize yet. It seems like the only way to inject the logic into the corporation would be to try to pass methods instead of values as some of the options or to monkey patch the Corporation class

Passing methods instead of values (such as `float_percent`) aren't flexible enough to work without making significant modifications to Corporation and putting defaults in the Base game and at that point I believe it would make it more complex overall.

I'm not comfortable with casually using monkey patching in general and in this case I don't know how that would affect the server side or client side code; I'm not sure if it could cause side effects with other games. 

That's why I've kept it as a game-specific subclass and some of the other subclasses and overrides that you've commented on elsewhere are done to support use of this game-specific class

There is an argument that the escrow functionality can be moved into the base class and I agree that it can be done, but I don't think it's time yet and that it'd be better to wait until we have another game or two that also uses "escrow" accounts for corporations to implement that in a general way. 
I know that 18SY (Original rules) keeps *shares* in escrow which get redeemed at market value at time of escrow release, so a static escrow balance wouldn't work there